src/orb.yml.hbs

```diff
@@ -132,6 +132,31 @@ jobs:
           "The name of the service to update. If undefined, we assume `family` is the name of both the service and task definition."
         type: string
         default: ""
+      deployment-controller:
+        description:
+          "The deployment controller to use for the service. Defaulted to ECS"
+        type: string
+        default: "ECS"
+      application-name:
+        description:
+          "The name of an AWS CodeDeploy application."
+        type: string
+        default: ""
+      deployment-group-name:
+        description:
+          "The name of an AWS CodeDeploy deployment group."
+        type: string
+        default: ""
+      container-name-for-code-deploy:
+        description:
+          "The name of the container for CodeDeploy."
```

Comment:
I would like to add some more details to the description to clarify the purpose of this parameter, like this:

```
description: |
  The name of the container to be load-balanced via AWS CodeDeploy.
  Only effective when the deployment-controller parameter value is "C
ODE_DEPLOY".
```